### DISCUSSION TITLE
Delete Reindexed Indices

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#13759] [Unsupported Versions] [answered]
Hello,

How do I remove these reindexed indices listed below?  Running the so-elastic-clear did not remove these reindexed indices that are in yellow state.

![image](https://github.com/user-attachments/assets/d894fb4b-18ea-48d5-b6c4-63ec72b8edba)

**Comment by cm-ops** [comment, marked as answer]
You could do it from Kibana > Stack Management > Index Management or use the delete API. Something like the following `sudo so-index-list | grep reindexed | awk '{print $3}' | xargs -I % so-elasticsearch-query % -XDELETE`
